content/en/docs/release_notes/dickens.md

```diff
@@ -0,0 +1,32 @@
+---

```

Comment:
Once the wicklow release notes are approved, we want them added to the v5 docs too since this should include all the up-to-date release notes.

Reply:
Added.